css_def/def.go

```diff
@@ -0,0 +1,48 @@
+package css_def
```

Comment:
I'll be deleting, we don't need it anymore, we are using sourcegraph's default formatter.